### ISSUE TITLE
Is this impacted by the deprecation of the v1 filtered realtime tweets API?

### ISSUE THREAD

**Issue by fareesh** [cvcio/rtaa-72#1]
Ref:

![image](https://user-images.githubusercontent.com/498161/199838555-abfdc16c-3713-4a7b-98f6-64c6cb3d859a.png)

If so, is there a straightforward migration process? 

After running the browser's request to the Stream service doesn't return a response and hangs indefinitely

**Comment by andefined** [comment]
@fareesh Yes, most probably this affects the streamer and the account [classifier](https://github.com/cvcio/rtaa-classifier) (v2 doesn't include favorites), although I am still able to use this endpoint (not sure why). We have already created a [client](https://github.com/cvcio/twitter) that supports V2 API endpoints. The migration should not be very trivial, as there are limitations on how many rules you can use and not yet tested, nevertheless, we will start working on this as soon as possible.
